pages/reference/diagnostics.md

```diff
@@ -0,0 +1,67 @@
+# Check Descriptions
+
+As part of the diagnostics suite, you will find various checks that can be run on-device. Below is a description of each
+check and what each means or how to triage.
+
+A `check` in this context is defined as a function that returns a result (good/bad status plus some descriptive text), whereas a
+command is simply a data collection tool without any filtering or logic built in). Checks are intended to be used by
+everyone, while typically command output is used by support/subject matter experts and power users.
+
+### check_resin1x
+#### Summary
+As of May 1, 2019, `resinOS 1.x` has been deprecated. These OSes are now unsupported. For more information, read our
+blog post: https://www.balena.io/blog/all-good-things-come-to-an-end-including-balenaos-1-x/
+
+#### Triage
+Upgrade your device to `balenaOS 2.x` (contact support).
+
+### check_under_voltage
+#### Summary
+Often seen on Raspberry Pi devices, these kernel messages indicate that the power supply is insufficient for the device and any peripherals that might be attached. These errors also precede seemingly erratic behavior.
+
+#### Triage
+Replace the power supply with a known-good supply (typically supplying at least 2.5V).
```

Comment:
2.5V is wrong, it should rather be 5V at >2.5A

Reply:
oops :man_facepalming: i swear i googled this too..